openhands/resolver/resolve_issue.py

```diff
@@ -344,6 +344,14 @@ async def resolve_issue(
         issue_numbers=[issue_number], comment_id=comment_id
     )
 
+    if not issues:
+        raise ValueError(
+            f'No issues found for issue number {issue_number}. Please verify that:\n'
+            f'1. The issue/PR #{issue_number} exists in the repository {owner}/{repo}\n'
+            f'2. You have the correct permissions to access it\n'
+            f'3. The repository name is spelled correctly (case-sensitive)'
```

Comment:
Not relevant, just wondering: is the repo name indeed case-sensitive here, or did the LLM just insert a common phrase? It doesn't seem so when accessed on the web (`https://github.com/All-Hands-AI/openhands/issues/5450` works) 🤷